core/build.gradle

```diff
@@ -74,6 +74,7 @@ dependencies {
     }
 
     compile "net.java.dev.jna:jna-platform:5.3.1"
+    compile "org.yaml:snakeyaml:1.21"
```

Comment:
this will make it a public dependency, something we don't have to have. Please make it shaded (see other shaded deps)